client/src/main/java/io/pravega/client/control/impl/ModelHelper.java

```diff
@@ -377,6 +379,17 @@ public static final NodeUri decode(final PravegaNodeUri uri) {
         return Controller.ScopeInfo.newBuilder().setScope(scope).build();
     }
 
+    public static final StreamInfo createStreamInfo(final String scope, final String stream,
```

Comment:
Can you have the other method you overloaded call into this one?

Reply:
Yeah, that makes sense. Changed it.